### PR TITLE
Add Table of Contents and what we own

### PR DESCRIPTION
**Pull request by s-rubenstein**
This adds the table of contents with short descriptions for each file, and a document for what services we own.

### CODE REVIEW

File: identity/What-We-Own.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# What repositories we manage
+
+## Current owned services in alphabetical order
+| Name | Description |
+| ---- | ----------- |
+| [Bard](https://github.com/DataBiosphere/bard) | Bard is a service which manages our integrations with metrics services. As of 6/24/2020, this is only Mixpanel. |
+| [Bueller](https://github.com/DataBiosphere/terra-ui/blob/dev/integration-tests/Bueller.md) | Bueller is a service which hosts our integration tests for the Terra UI as an application, with one endpoint for each integration test. |
+| [Calhoun](https://github.com/DataBiosphere/calhoun) | Calhoun is a service which generates read-only views of notebooks for previews. It does not require a running cluster, and spits out the notebook file as markdown. |
```
Comment (breilly2) [inline identity/What-We-Own.md]:
```suggestion
| [Calhoun](https://github.com/DataBiosphere/calhoun) | Calhoun is a service which generates read-only views of notebooks for previews. It does not require a running cluster, and spits out the notebook file as HTML. |
```

File: identity/What-We-Own.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+| [Job Manager](https://github.com/DataBiosphere/job-manager) | Job manager is a UI and API service that is ued by users of the Terra platform to manage and inspect in progress, finished, or errored batch jobs. |
+| [Lyle](https://github.com/DataBiosphere/lyle) | Lyle is a service to manage allocating temporary google service accounts for our test users |
```
Comment (breilly2) [inline identity/What-We-Own.md]: I'll take a stab at describing rex:
```suggestion
| [Lyle](https://github.com/DataBiosphere/lyle) | Lyle is a service to manage allocating temporary google service accounts for our test users |
| [Rex](https://github.com/DataBiosphere/rex) | Rex is a service for collecting NPS survey responses. |
```

Comment (breilly2) [inline identity/What-We-Own.md]: Nit: "for our test users" sounds a little like it creates service accounts for existing users (like pets). Maybe something like this?
```suggestion
| [Lyle](https://github.com/DataBiosphere/lyle) | Lyle is a service to manage allocating temporary google service accounts for use as test users |
```

**Review by Shakespeared — APPROVED** [review]
thumbs up

**Review by breilly2 — COMMENTED** [review]
A couple of minor suggestions, plus an attempt to describe rex which we also own.

**Pull request merged**